react/src/Components.js

```diff
@@ -17,10 +17,14 @@ export const Link = ({to, env = true, bold = true, id = null, children}) => {
     </>
 }
 
-export const HorizontalLine = ({top = "0", bottom = "0", thick = false, color = null }) => {
-    return color ? 
-           <div style={{background:color,height:thick ? "2px" : "1px",marginTop:top,marginBottom:bottom}}></div> :
-           <div className="fgbg" style={{height:thick ? "2px" : "1px",marginTop:top,marginBottom:bottom}}></div>
+export const HorizontalLine = ({top = "0", bottom = "0", thick = false, color = null, dotted = false }) => {
+    if (!Str.HasValue(color)) color = Styles.GetForegroundColor();
+    const thickness = thick ? "2px" : "1px";
+    const style = dotted ? "dotted" : "solid";
```

Comment:
I might call this variable `borderStyle` or `maybeDotted` or `dottedOrSolid`.

Reply:
Sounds good.